text/0007-ownership-custodians.md

```diff
@@ -0,0 +1,113 @@
+# Ownership Custodians
+
+The health of <https://rubygems.org> is important for the community. Having a large number of essentially abandoned gems contributes negatively towards the usability and impression of RubyGems, and we would like to improve the mechanisms for dealing with this problem.
+
+Developers who want to release new Ruby libraries are often hampered by the lack of good available gem names. There is no formal process for reusing gem names which are effectively abandoned. Informal processes often break down due to lack of trust or uncontactable gem owners. This can create friction for developers as they then have to consider alternative names which are less ideal.
+
+For users who search RubyGems to find relevant projects, they are often presented with many options which have not been updated for many years. This presents a bad impression of the Ruby community as "unmaintained". This problem will only get worse over time if no effort is made to address the underlying reasons.
+
+We propose to introduce gem custodians and a process for approving ownership requests where the original owner is unresponsive or uncooperative. This would be an accountable process to deal with abandoned gem namespace and enable valuable gem namespace, which is effectively abandoned, to be reused in a way which is ultimately positive for the Ruby community.
```

Comment:
I am not sure I understand the importance of a new gem custodians group. rubygems admins should reserve the right to make the final decision.
IMO, the issue with a new group would be ensuring their motives align with rubygems. Further given that most likely it will be a volunteer position, they may not have the bandwidth of response/resolution within acceptable deadlines (which is currently offered by rubygems support).